logger/CMakeLists.txt

```diff
@@ -21,34 +21,25 @@ add_subdirectory(${CMAKE_BINARY_DIR}/spdlog-src
   EXCLUDE_FROM_ALL
 )
 
-set(MODULE_LIBS spdlog_header_only)
-set_target_properties(spdlog_header_only PROPERTIES EXPORT_NAME "positioning_systems_api_spdlog_header_only")
-
-add_library(${LIB_NAME}
+add_library(${LIB_NAME} SHARED
   src/logger.cpp
 )
 
 target_include_directories(${LIB_NAME} PUBLIC
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include/>
+  $<BUILD_INTERFACE:${CMAKE_BINARY_DIR}/spdlog-src/include/>
   $<INSTALL_INTERFACE:include/>
 )
 
 target_link_libraries(${LIB_NAME} ${MODULE_LIBS})
 
-install(DIRECTORY include/
+install(DIRECTORY include/ ${CMAKE_BINARY_DIR}/spdlog-src/include/
```

Comment:
I don't think it is necessary to install spdlog headers. Do you agree? @afvillamil231

Reply:
I think it is as you are using the headers in your examples. If you don't want the users to call the headers then you can remove their installation.

Reply:
you basically do
```C++
#include "follow_me_driver/follow_me_driver.hpp"
```
which at the same time does
```C++
#include "logger/logger.hpp"
```
which at the same time does
```C++
#include <spdlog/fmt/ostr.h>
#include <spdlog/spdlog.h>
```
you can maybe try removing those two last lines from logger.hpp